templates/docs/patterns/slider.md

```diff
@@ -15,9 +15,6 @@ when you want to provide a numeric representation of the slider value, as well a
 View example of the slider pattern
 </a></div>
 
-<span class="p-label--updated">Updated</span> In version 2.6.0 Vanilla framework added slider styling as default for
```

Comment:
I would keep this one, as it's something related to upgrading Vanilla. Worth keeping (like the deprecated labels)